lib/rucio/core/rule.py

```diff
@@ -3526,7 +3541,7 @@ def archive_localgroupdisk_datasets(scope, name, *, session: "Session", logger=l
 
 @policy_filter
 @read_session
-def get_scratch_policy(account, rses, lifetime, *, session: "Session"):
+def get_scratch_policy(account: str, rses: Sequence[dict[str, Any]], lifetime: int, *, session: "Session") -> int:
```

Comment:
1. `account` is `InternalAccount`
 2. `lifetime` is `Optional[int]`

Reply:
If I change `lifetime` to `Optional[int]`, then the linter complains that the return type should also be `Optional[int]`. I can rewrite this to only return `lifetime` if it's not `None`, but wanted to check first if that's correct. I think in this case `Optional[int]` might be the correct return type? 

`if not has_account_attribute(account=account, key='admin', session=session) and lifetime is None` and if none of the RSEs are SCRATCHDISKs, then `lifetime` would be `None`

Reply:
Well observed. Yes, I believe returning `None` is a valid. All the callers of this functions handle it properly.